.github/PULL_REQUEST_TEMPLATE/develop-template.md

```diff
@@ -0,0 +1,16 @@
+Originator(s):
+
+Summary (include the keyword ['closes', 'fixes', 'resolves'] and issue number):
+
+Describe any changes made to the namelist:
+
+List all files eliminated and why:
+
+List all files added and what they do:
+
+List all existing files that have been modified, and describe the changes: 
+(Helpful git command: `git diff --name-status development...<your_branch_name>`)
+
+List any test failures:
+
+Is this a science-changing update? New physics package, algorithm change, tuning changes, etc?
```

Comment:
I would suggest the bullet should be something like:  `Is this an answer changing PR?  If so, is it a new physics package, algorithm change, tuning changes, etc?`